src/sql/src/session/vars.rs

```diff
@@ -2776,6 +2894,25 @@ impl Value for Option<mz_repr::Timestamp> {
     }
 }
 
+impl Value for Option<i32> {
+    const TYPE_NAME: &'static str = "optional signed integer";
+
+    fn parse(input: VarInput) -> Result<Option<i32>, ()> {
+        let s = extract_single_value(input)?;
+        match s {
+            "" => Ok(None),
+            _ => <i32 as Value>::parse(VarInput::Flat(s)).map(Some),
+        }
+    }
+
+    fn format(&self) -> String {
+        match self {
+            Some(s) => s.format(),
+            None => "".into(),
+        }
+    }
+}
```

Comment:
Does this work as intended? Paging @mjibson and/or @jkosh44. I believe the way we do this for other numeric params that have defaults is to follow PostgreSQL and declare `0` as the default value. The empty string bit seems sketchy. 

Looking a bit deeper, huh, I guess we do this for `cluster_replica` and `mock_audit_event_timestamp`? Seems worth digging a bit though; neither `cluster_replica` nor `mock_audit_event_timestamp` are controlled by LD, so it's a bit different.

Reply:
Ill investigate locally

Reply:
I'm not very familiar with this part of vars, but using an empty string seems inline with other `Option` var types.

Reply:
I tested with LD and it worked as expected!